docs/modules/ROOT/pages/mcp.adoc

```diff
@@ -0,0 +1,33 @@
+= Model Context Protocol
+
+LangChain4j supports the Model Context Protocol (MCP) to communicate with
+MCP compliant servers that can provide and execute tools. General
+information about the protocol can be found at the
+https://modelcontextprotocol.io/[MCP website]. More detailed information can
+also be found in the https://docs.langchain4j.dev/tutorials/mcp[LangChain4j
+documentation], this documentation focuses on features that Quarkus provides
+on top of the upstream module. For an example project that uses MCP, see
+https://github.com/quarkiverse/quarkus-langchain4j/tree/main/samples/mcp-tools[mcp-tools]
+project in the `quarkus-langchain4j` repository.
+
+// TODO: double-check the https://docs.langchain4j.dev/tutorials/mcp link
+// works after the langchain4j docs are updated, as well as the mcp-tools project link
```

Comment:
Has this been done?

Reply:
For the upstream link, yes
For quarkus-langchain4j docs, obviously not, but I think we can quite safely remove this comment now